src/KeyUnit.cpp

```diff
@@ -116,6 +126,29 @@ bool KeyUnit::disableKey(const QString & name )
     return found;
 }
 
+bool KeyUnit::killKey( QString & name ) {
+    typedef list<TKey *>::const_iterator I;
+    for( I it = mKeyRootNodeList.begin(); it != mKeyRootNodeList.end(); it++)
```

Comment:
Yeah, since we've adopted C++11 you can now use the `auto` keyword to avoid the `typedef`s... :grinning: